### PR TITLE
Ts/con 456 dynamic form

### PR DESCRIPTION
**Pull request by MkMan**

### CODE REVIEW

File: src/compositions/DynamicForm/DynamicForm.tsx [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+// };
+
+// DynamicForm.defaultProps = {
+//   children: undefined,
+//   className: undefined,
+//   defaultValues: undefined,
+//   formSchema: undefined,
+//   onSubmit: undefined,
+//   theme: undefined,
+// };
```
Comment (kevin-ho87) [inline src/compositions/DynamicForm/DynamicForm.tsx:134]: Should these comments be removed?

Comment (MkMan) [reply src/compositions/DynamicForm/DynamicForm.tsx:134]: Yes. Yes they definitely should be 😄  Thank you so much for picking up on that 🙏

File: src/compositions/DynamicForm/wrappers/TextField.tsx [hunk 2]
```diff
@@ -1,9 +1,14 @@
-/* eslint-disable react/prop-types */
-import React from 'react';
+import React, { VFC } from 'react';
```
Comment (kevin-ho87) [inline src/compositions/DynamicForm/wrappers/TextField.tsx:1]: what is `VFC` ?

Comment (MkMan) [reply src/compositions/DynamicForm/wrappers/TextField.tsx:1]: Void Function Component, a Function Component that can't have `children`

https://www.mydatahack.com/using-react-vfc-instead-of-react-fc/

Comment (kevin-ho87) [reply src/compositions/DynamicForm/wrappers/TextField.tsx:1]: such a dark name. :P

**Review by kevin-ho87 — APPROVED** [review]

File: src/compositions/DynamicForm/DynamicForm.tsx (hunk 1 above)
Comment (hassanraza0054) [reply src/compositions/DynamicForm/DynamicForm.tsx]: I wonder if we're keeping these now to see if any bugs come out with the Typescript conversion and we can revert back?

File: src/compositions/DynamicForm/wrappers/TextField.tsx (hunk 2 above)
Comment (hassanraza0054) [reply src/compositions/DynamicForm/wrappers/TextField.tsx:1]: https://github.com/yannickcr/eslint-plugin-react/issues/2913

@kevin-ho87 from what i can tell, its a Type that allows you to define a component without the **children** prop. Correct me if im wrong or if there is more to it!

**Pull request merged**